frontend/src/Login/CardTextInput.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { TextInput } from 'react-native';
+import { CardView } from '../common/components';
+
+const style = {
+  height: 40,
+  margin: 5,
+  paddingStart: 10,
+  fontSize: 14,
+};
+
+function CardTextInput({
```

Comment:
I'm not sure why do we use so much `Card`-s in the naming :D but you can just call it InputField or something like that. Of course this is fine too:)

Reply:
I made a new style for TextInput based on card style,
so I call it CardTextInput, If we have a better name for it, I will use it.